src/storage/main.cpp

```diff
@@ -487,13 +490,43 @@ static void signalHandler(sdeventplus::source::Signal& source,
     source.get_event().exit(EXIT_SUCCESS);
 }
 
+static void showUsage(char* appName)
+{
+    fprintf(stderr, R"(Usage: %s [options]
+Options:
+  -v, --verbose  Enable output debug messages.
+  -h, --help     Show this help
+)",
+            appName);
+}
+
 /**
  * @brief Application entry point
  *
  * @return exit status
  */
-int main()
+int main(int argc, char** argv)
 {
+    const struct option opts[] = {
+        {"verbose", no_argument, nullptr, 'v'},
+        {"help", no_argument, nullptr, 'h'},
+        {nullptr, 0, nullptr, '\0'}};
```

Comment:
You definitely forgot the `--disable-debug-log` option everywhere, not just in the usage help.

Reply:
Then an update to the commit log is required.

Reply:
fixed